### PR TITLE
First draft of new archive lesson on data

### PR DESCRIPTION
**Pull request by JennyBunn**
This is the first time I have done this. I have no idea if I have done it right, but I am trying to change the wording of this lesson as part of the work to revise it for archivists.

### CODE REVIEW

**Comment by drjwbaker** [comment]
@JennyBunn Thanks so much for your hard work. I'll review when I get a minute and let back with comments.

File: _episodes/02-jargon-busting.md [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
 questions:
-- What terms, phrases, or ideas around code or software development have you come across and perhaps feel you should know better?
+- What sort of data do you work with? 
+- What do you do with it? 
+- What tools do you use to help you?
 objectives:
-- Explain terms, phrases, and concepts associated with terms, phrases, or ideas around code or software development in libraries
-- Compare knowledge of these terms, phrases, and concepts
-- differentiate between these terms, phrases, and concepts
+- Get participants thinking in terms of data and recognising that they do have some
```
Comment (drjwbaker) [inline _episodes/02-jargon-busting.md]: Align with Bloom's https://swcarpentry.github.io/instructor-training/19-lessons/

Perhaps:

> Recognize that they work with data

**Review by drjwbaker — CHANGES_REQUESTED** [review]
Objective 1 needs to align with Bloom's https://swcarpentry.github.io/instructor-training/19-lessons/ Perhaps:

> Recognize that they work with data

The last bit `and which of the problems raised what we will be learning will help to address.` doesn't make sense to me.

**Comment by JennyBunn** [comment]
Okay so I need to go back and change those, do another commit and then another pull request right?

**Comment by drjwbaker** [comment]
Yes. If you can't figure it out quickly, I'll merge what we have and then you can change the merged version.

**Comment by JennyBunn** [comment]
Thank you for your help.

**Pull request merged**